android/filament-android/src/main/java/com/google/android/filament/RenderableManager.java

```diff
@@ -203,6 +205,18 @@ public Builder blendOrder(@IntRange(from = 0) int index,
             return this;
         }
 
+       /**
+         * Sets whether the blend order is global or local to this Renderable (by default).
```

Comment:
Maybe put the "(by default)" right after "local":

```suggestion
         * Sets whether the blend order is global or local (by default) to this Renderable.
```

Same suggestion elsewhere.